src/core/TSDB.java

```diff
@@ -12,59 +12,17 @@
 // see <http://www.gnu.org/licenses/>.
 package net.opentsdb.core;
 
-import java.io.File;
```

Comment:
So, it looks like you did the "auto format" in IntelliJ, I've been tempted to commit this myself. Would it be possible to isolate just the changes relevant to your new plugin type?

I love the concept and would be happy to get this merged if we can remove the unrelated changes.